notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1376 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #65.        )
</pre><p>I know you are demonstrating profiling, but it's a little strange to not save the model after training it, don't you think?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>Added model.save()</p>